.golangci.yml

```diff
@@ -4,6 +4,22 @@ linters:
     - unparam
     - revive
     - depguard
+    - misspell
+    - ineffassign
+    - typecheck
+    - varcheck
+    - unused
```

Comment:
It seems we have listed some lint that is open by default? But it shouldn't matter.

Reply:
yes indeed